flink-connectors/flink-connector-hbase-base/src/main/java/org/apache/flink/connector/hbase/table/HBaseConnectorOptions.java

```diff
@@ -87,6 +90,17 @@
                                     + "Can be set to '0' to disable it. Note, both 'sink.buffer-flush.max-size' and 'sink.buffer-flush.max-rows' "
                                     + "can be set to '0' with the flush interval set allowing for complete async processing of buffered actions.");
 
+    public static final ConfigOption<String> SINK_DELETE_MODE =
+            ConfigOptions.key("sink.delete.mode")
+                    .stringType()
```

Comment:
> Wouldn't it be better to use `enumType` here since the option is an enum? Then we can also improve the docs on the option by documenting the enum itself (which should be PublicEvolving), and avoid duplicating the entire thing as string constants.

Thanks for your review. Because the available values contain '-' characters, the enum type is similar to 'XXX_XXX', so it needs to be converted. Please take a look JsonFormatOptionsUtil.getTimestampFormat().